### ISSUE TITLE
Some tags not collected

### ISSUE THREAD

**Issue by klase** [tizmagik/react-head#57]
When trying to add the following Meta tags (3.0.0-1):

```
<Title>{title}</Title>
<Meta name="description" content="The title" />
<Meta property="og:description" content="Some description" />
<Meta property="og:image" content="http://example.com/image.jpg" />
<Meta property="og:title" content="OG title" />
<Meta property="og:sometag" content="someValue" />
```

Only the first and last one is actually collected and rendered.

**Comment by TrySound** [comment]
What sense have `property` attribute in `meta` tag? Do you have any spec references? For now meta tags are distinguished only by `name` attribute, so in your case there're two meta with `"description"` name and `undefined` name.

**Comment by klase** [comment]
It is used in the open graph protocol (http://ogp.me/) which was originally created by Facebook but is now managed by the Open Web Foundation. 

Without it all link shares on facebook and twitter etc would be blank.

**Comment by TrySound** [comment]
Okay, I think distinguishing by `name || property` should work.

**Comment by tizmagik** [comment]
I think this was addressed in #58 so closing. @klase please re-open or create a new issue if you see any problems! Thanks!
